README.md

```diff
@@ -3,10 +3,10 @@
 The RDF/JS SHACL-UI Task Force is a focus group of the RDF/JS Community Group.
 
 The objective is to enhance the SHACL ontology to cater specifically to the UI use cases for application development and data exploration.
-It covers reading and writing RDF data.
+It covers viewing/displaying, writing and querying RDF data.
 Interfaces for reusable UI components should be established.
-To achieve this, we aim for implementations to rely solely on the SHACL ontology and the UI extension, enabling them to select UI components and generate layouts.
-The defined interfaces will enable seamless communication between implementations and UI components, enabling efficient reading and writing of RDF data.
+To achieve this, we aim for implementations to rely on the SHACL ontology and the UI extension, enabling them to select UI components and generate layouts.
```

Comment:
I wrote "solely" because it should require only two ontologies, SHACL and the new one, which we will work on. Do you think other ontologies will be involved, or why do you want to remove it?

Reply:
Why should an implementation be **required** to rely only on these 2 ontologies ? why can't other ontologies be involved ? What if there is e.g. a third extension of the ontology we will be developing ? why closing the world ? Since you are asking, yes I can think about http://data.sparna.fr/ontologies/volipi/index-en.html.

Reply:
Maybe the wording could be improved, but I read it like this:

An implementation must use SHACL + new ontology for the logic to select UI components and generate the layouts. Other ontologies can be used for additional features like controlling the component behavior or for defining new layouts.

The constraint is required to have interoperability on the controller level.

But that's just my opinion. The subgroup has to decide in the end. Changes are welcome if you agree with me, and it's only a problem with the wording. I'm also ok with your changes and moving further discussions to the subgroup.